services/director/docker/entrypoint.sh

```diff
@@ -8,6 +8,17 @@ INFO="INFO: [$(basename "$0")] "
 WARNING="WARNING: [$(basename "$0")] "
 ERROR="ERROR: [$(basename "$0")] "
 
+# Read self-signed SSH certificates (if applicable)
+#
+# entrypoint.sh is run as root. For the local deployment of oSparc,
+# we need to point the director to the self-signed certificate in order to
+# connect to the local docker-container-registry. While the certificates
+# are handled in the docker-compose file for the local deployment in the ops-repo,
+# we need to call 'update-ca-certificates' in order to read them (as root!)
+# This script is the proper place for this necessary call. If the deployment is non-local
+# nothing will happen.
+update-ca-certificates
```

Comment:
Maybe I would change the very long comment there to something like:
```
In case the director must access a registry in a secure way using non-standard  certificates (e.g. such as self-signed certificates), this call is needed.
```

I do not think that anything with regards to "ops-repo" or so would be necessary. Especially since the simcore repo does not know anything about the other repo.